projects/step-frontend/src/lib/modules/timeseries/pages/execution-page/ts-execution-page.component.ts

```diff
@@ -118,33 +147,8 @@ export class ExecutionPerformanceComponent implements OnInit, OnDestroy, OnChang
       activeTimeRange: timeRangeOptions[timeRangeOptions.length - 1],
       contextualFilters: { ...urlParams, eId: this.executionId },
       showContextualFilters: false,
-      filterOptions: [
-        {
-          label: 'Status',
-          attributeName: 'rnStatus',
-          type: FilterBarItemType.OPTIONS,
-          textValues: [
-            { value: 'PASSED' },
-            { value: 'FAILED' },
-            { value: 'TECHNICAL_ERROR' },
-            { value: 'INTERRUPTED' },
-          ],
-          isLocked: true,
-        },
-        {
-          label: 'Type',
-          attributeName: 'type',
-          type: FilterBarItemType.OPTIONS,
-          textValues: [{ value: 'keyword' }, { value: 'custom' }],
-          isLocked: true,
-        },
-        {
-          label: 'Name',
-          attributeName: 'name',
-          type: FilterBarItemType.FREE_TEXT,
-          isLocked: true,
-        },
-      ],
+      activeFilters: pageFilters,
+      filterOptions: JSON.parse(JSON.stringify(pageFilters)),
```

Comment:
In case if you want to have a copy of the default object, it make sense to create a separate method, that will construct and return filters.
```
activeFilters: this.createFilters(),
filterOptions: this.createFilters()
```
Both fields will have similar values, but with completely different references.
I think it's better than parse/stringify invocation